### PR TITLE
fix command results to context

### PR DESCRIPTION
**Pull request by anara123**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
- fixed when passing multiple indicators to CommandResults it was not converting properly to context
- fixed UpdateDate to UpdatedDate in Domain/WHOIS according to Standard Context docs
- dbot_score is now mandatory for all the reputation classes

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [x] Yes
       - no one is using the new classes right now, it should be fine
   - [ ] No

## Must have
- [x] Tests
- [ ] Documentation 

## Demisto Partner?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

**Comment by anara123** [comment]
@fvigo FYI

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 1]
```diff
@@ -2336,14 +2292,31 @@ def to_context(self):
             if self.positive_detections:
                 domain_context['PositiveDetections'] = self.positive_detections
 
-            if self.whois:
-                domain_context['WHOIS'] = self.whois.to_context()
+            if self.registrar_name or self.registrar_abuse_email or self.registrar_abuse_phone:
+                domain_context['Registrar'] = {
```
Comment (fvigo) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:2296]: According to https://xsoar.pan.dev/docs/integrations/context-standards#domain  , Domain.Registrar doesn't exist. Only Domain.WHOIS.Registrar exist.

File: Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.yml [hunk 2]
```diff
@@ -1045,7 +1045,8 @@ script:
     - contextPath: Domain.Name
       description: The domain name.
       type: String
-  dockerimage: demisto/python3:3.7.2.214
+  dockerimage45: demisto/python3:3.7.2.214
```
Comment (ronykoz) [inline Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.yml:1048]: why do we need this one?
did you add something special for 3.8? if not there is no reason for this

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 3]
```diff
@@ -2125,7 +2147,7 @@ class CVE(Indicator):
         :return: None
         :rtype: ``None``
         """
-        CONTEXT_PATH = 'URL(val.Data && val.Data == obj.Data)'
+        CONTEXT_PATH = 'CVE(val.ID && val.ID == obj.ID)'
```
Comment (ronykoz) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:2150]: why don't we have dbotscore for CVE ?

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 4]
```diff
@@ -2430,24 +2412,38 @@ def to_context(self):
 
         if self.indicators:
             for indicator in self.indicators:
-                outputs.update(indicator.to_context())
+                context_outputs = indicator.to_context()
+                for key in context_outputs.keys():
+                    value = context_outputs[key]
```
Comment (ronykoz) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:2417]:
```suggestion
                for key, value in context_outputs.items():
```

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 5]
```diff
@@ -2430,24 +2412,38 @@ def to_context(self):
...
-            outputs_key = '{}(val.{} == obj.{})'.format(self.outputs_prefix, self.outputs_key_field, self.outputs_key_field)
-            outputs.update({
-                outputs_key: self.outputs
-            })
+            if self.outputs_prefix and self.outputs_key_field:
+                # if both prefix and key field provided then create DT key
+                outputs_key = '{}(val.{} == obj.{})'.format(self.outputs_prefix, self.outputs_key_field, self.outputs_key_field)
+                outputs[outputs_key] = self.outputs
+            elif self.outputs_prefix:
+                outputs_key = '{}'.format(self.outputs_prefix, self.outputs_key_field)
```
Comment (ronykoz) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:2442]:
```suggestion
                outputs_key = '{}'.format(self.outputs_prefix)
```

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 6]
```diff
@@ -2430,24 +2412,38 @@ def to_context(self):
...
             if not self.raw_response:
                 raw_response = self.outputs
 
-            outputs_key = '{}(val.{} == obj.{})'.format(self.outputs_prefix, self.outputs_key_field, self.outputs_key_field)
-            outputs.update({
-                outputs_key: self.outputs
-            })
+            if self.outputs_prefix and self.outputs_key_field:
+                # if both prefix and key field provided then create DT key
+                outputs_key = '{}(val.{} == obj.{})'.format(self.outputs_prefix, self.outputs_key_field, self.outputs_key_field)
```
Comment (ronykoz) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:2439]: would be nicer i think
```suggestion
                outputs_key = '{0}(val.{1} == obj.{1})'.format(self.outputs_prefix, self.outputs_key_field)
```

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py (hunk 3 above)
Comment (anara123) [reply Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:2150]: CVEs has no DBotScores. It is not an indicator like IP.
There was an old discussion whether we want CVEs have DBot Score and at the end decided not to have.
@michalgold correct me if I am wrong

File: Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.yml (hunk 2 above)
Comment (anara123) [reply Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.yml:1048]: The validation fails on it.

**Comment by anara123** [comment]
```
Autofocus Query Samples, Sessions and Tags Test Playbook failed with error/s
Playbook Autofocus Query Samples, Sessions and Tags Test Playbook has failed:
- Task ID: 13
  Command: !autofocus-search-sessions query="{\"operator\":\"any\",\"children\":[{\"field\":\"session.sha256\",\"operator\":\"is\",\"value\":\"4f79697b40d0932e91105bd496908f8e02c130a0e36f6d3434d6243e79ef82e0\"},{\"field\":\"session.sha256\",\"operator\":\"is\",\"value\":\"7e93723c0c34ef98444e5ce9013fef220975b96291a79053fd4c9b3d3550aeb3\"}]}" max_results="30" file_hash="" domain="" ip="" url=""
  Body:
Search operation failed: Server Error

- Task ID: 8
  Command: Executing conditions:
Label: yes, Condition: [null Is not empty OR null Is not empty]
  Body:
The output [] didn't satisfy any of the conditions of task 8 (31)

- Task ID: 8
  Command: Executing conditions:
Label: yes, Condition: [null Is not empty OR null Is not empty]
  Body:
The output [] didn't satisfy any of the conditions of task 8 (31)

Failed: playbook: Autofocus Query Samples, Sessions and Tags Test Playbook with integration(s): AutoFocus V2 failed
[2020-05-10 10:25:13.674292] ------ Test playbook: Autofocus Query Samples, Sessions and Tags Test Playbook with integration(s): AutoFocus V2 end ------
```
The build failed on this playbook. It failed because of server error on Autofocus side. It passed in master. It passed after I re-run it in the failed instance.
The failure is not related to the changes in the PR

**Pull request merged**